Sources/NFCPassportReader/NFCPassportModel.swift

```diff
@@ -43,6 +43,27 @@ public class NFCPassportModel {
         return name.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
     }()
     
+    
+//  adding capability to read different data such as residence address, place of birth, phone number from data group 11
+    
+    ///place of birth
+    public private(set) lazy var placeOfBirth : String = {
+        guard let dg11 = dataGroupsRead[.DG11] as? DataGroup11 else { return "State/City not available" }
```

Comment:
```suggestion
        guard let dg11 = dataGroupsRead[.DG11] as? DataGroup11 else { return "State/City not available" }
```

I would not go for the mentioned default strings, as the only way to make sure that the fields aren't set from the outside would be to check against the specific strings such as "State/City not available".
As a developer integrating this library, I would expect the field to be either nil or empty. Just my two cents :)